pandas/tests/groupby/test_groupby.py

```diff
@@ -5777,6 +5769,81 @@ def test_agg_over_numpy_arrays(self):
 
         assert_frame_equal(result, expected)
 
+    def test_cummin_cummax(self):
+        # GH 15048
+        # Test with different dtypes
+        for dtype in [np.int32, np.float32, np.float64]:
```

Comment:
also test with ``object`` dtype (which we don't expect to work, I *think* it should raise). but we can handle this in another issue.